client/login/wp-login/index.jsx

```diff
@@ -104,12 +118,16 @@ class Login extends React.Component {
 				</a>;
 		}
 
-		const goBackLink = ! magicLoginView && <a
-			href="#"
-			key="back-link"
-			onClick={ this.goBack }>
-				<Gridicon icon="arrow-left" size={ 18 } /> { this.props.translate( 'Return' ) }
-			</a>;
+		let goBackLink;
+		if ( this.state.loaded && window.history.length > 1 ) {
```

Comment:
We are server rendering this component. Node doesn't have access to `window` object. We can use
```js
window && window.history.length > 1
```
to make sure it doesn't throw error on the server.